conda_forge_tick/migrators/version.py

```diff
@@ -361,6 +361,19 @@ def _fmt_error_message(errors, version):
     return msg
 
 
+def check_version_matches_cadence(new_version: str, version: str, bot):
+    """This function takes two versions strings and compare them by using a given pattern."""
+    if "version_pattern" not in bot or not bot["version_pattern"]:
+        return False
+    pattern = bot.get("version_pattern")
+    if pattern is not None:
+        split_version = re.match(pattern, new_version)
+        # if version is greater then the new_version (~parsed) then we should not perform the PR
+        if version >= split_version.group():
```

Comment:
I think this is handled elsewhere

Reply:
yeah... but not using the pattern (We can set just the `re.match(pattern, new_version).group() >= version` inside the `version_filter` directly if you want -- but there is the problem with getting the key then)

Reply:
Here if `version >= the_new_one` then (it means that the current version is the last stable) we don't need to merge it.

Reply:
ahhh now I get what you've said, sorry... yeah there is no need for the comparasion here just an `if re.match(pattern, new_version).group() == new_version` should work well right ?